### DISCUSSION TITLE
Debugging 'in progress' events

### DISCUSSION THREAD

**Discussion by DrSpaldo** [blakeblackshear/frigate#3415] [General]
I've been getting a few false person detections (ie. BBQ) and have noticed that they are not displayed anywhere when you go into the live camera debug mode. I am not sure if they are supposed to be there when you turn on the options down the bottom?

**Comment by NickM-27** [comment]
If the false detection is in progress then it should be there with bounding boxes enabled. Example screenshots would be helpful to understand what exactly is going on

**Comment by DrSpaldo** [comment]
Having a second look, I am a little confused if it is actually Frigate or the link to HA being incorrect. It no longer says "in progress" when I look at Frigate, however, Home Assistant still has it as an active 'person'.

Screenshots attached as requested
![2022-07-03 screenshot (2)](https://user-images.githubusercontent.com/8526509/177019022-390eb833-d6f5-48fa-b9c8-f19c0d07b4e3.jpg)

![2022-07-03 screenshot (1)](https://user-images.githubusercontent.com/8526509/177019024-4b77d31f-7098-4f6e-b8df-047748b1caf5.jpg)

![2022-07-03 screenshot](https://user-images.githubusercontent.com/8526509/177019025-bf899a79-7d85-4a2f-80c4-adf357488905.jpg)

I think that my initial post is misleading as the box probably would be there, must be something between HA & Frigate?

**Reply by NickM-27** [reply]
You're likely experiencing https://github.com/blakeblackshear/frigate/issues/2927
